test_root/.jenkins-coveragerc

```diff
@@ -1,4 +1,8 @@
 [paths]
-source =
+build_jenkins_source =
```

Comment:
Sort of... The names of the groups can be anything, they aren't actually used by coverage or in the groovy scripts.  According the documentation:

> The names of the entries are ignored, you may choose any name that you like. The value is a lists of strings. When  combining data with the combine command, two file paths will be combined if they start with paths from the same  list.
>
> The first value must be an actual file path on the machine where the reporting will happen, so that source code can  be found. The other values can be file patterns to match against the paths of collected data, or they can be absolute or relative file paths on the current machine.